### PR TITLE
implemented slider selection behavior

### PR DESCRIPTION
**Pull request by MarekTalpas**
At last, I changed slider selection behavior. It should work as expected. However, I didn't made tests for it so far. Maybe, I will so during weekend.

### CODE REVIEW

**Comment by garySiladi** [comment]
In circumstances when there is fewer episodes than number of slots ( Daily Watch for example ) it doesn't work as intended.

File: src/components/slider/slider.js [hunk 1]
```diff
@@ -6,6 +6,36 @@ import SliderStructure from '../../structures/root';
 import './slider.css';
 
 class Slider extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  static getSelectorStyle(isSelected, selectedEpisode, data) {
+    const horizontalOffset = -320;
```
Comment (garySiladi) [inline src/components/slider/slider.js]: what do you say to an implementation of this function like:
```
const sliderMarginWidth = 115;
const sliderItemWidth = 320;
const offsetX = 0 - sliderItemWidth;
const homePageWidth = window.innerWidth - sliderMarginWidth;
const viewportHorizontalSlotNumber = Math.trunc(homePageWidth / sliderItemWidth);
const itemsToTheRightFromSelected = data.length - selectedEpisode;
const areMoreEpisodesThanSlots = data.length > viewportHorizontalSlotNumber;
const itemsBeforeSelectedEpisode =
  areMoreEpisodesThanSlots ? data.length - viewportHorizontalSlotNumber : 0;
return (viewportHorizontalSlotNumber > itemsToTheRightFromSelected ?
  itemsBeforeSelectedEpisode : selectedEpisode) * offsetX;
```

File: src/components/slider/slider.js [hunk 2]
```diff
@@ -6,6 +6,19 @@ import SliderStructure from '../../structures/root';
 import './slider.css';
 
 class Slider extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  static getHorizontalOffset(selectedEpisode, data) {
+    const sliderMarginWidth = 115;
+    const sliderItemWidth = 320;
+    const offsetX = 0 - sliderItemWidth;
```
Comment (varholak-peter) [inline src/components/slider/slider.js]: this variable is not needed you can do `sliderITemWidth * -1` in the place you are currently using offsetX

Comment (garySiladi) [inline src/components/slider/slider.js]: these are just abstractions really ... but we can modify

File: src/components/slider/slider.js [hunk 3]
```diff
@@ -21,20 +51,20 @@ class Slider extends React.Component { // eslint-disable-line react/prefer-state
       className,
       sliderTitle,
       selectedEpisode,
+      data,
```
Comment (garySiladi) [inline src/components/slider/slider.js]: like `data: newname`

**Pull request merged**